src/Core/src/PropertyMapper.cs

```diff
@@ -35,29 +39,51 @@ public PropertyMapper(params IPropertyMapper[]? chained)
 		protected virtual void SetPropertyCore(string key, Action<IElementHandler, IElement> action)
 		{
 			_mapper[key] = action;
+
 			ClearKeyCache();
 		}
 
+		// TODO: Remove in .NET10
 		protected virtual void UpdatePropertyCore(string key, IElementHandler viewHandler, IElement virtualView)
 		{
 			if (!viewHandler.CanInvokeMappers())
+			{
 				return;
+			}
 
-			var action = GetProperty(key);
-			action?.Invoke(viewHandler, virtualView);
+			if (MergedMappers.TryGetValue(key, out var action))
```

Comment:
Should this just call TryUpdatePropertyCore?

Reply:
Yeah, that's definitely an option. I'll do once we have CI results.